src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -404,29 +404,82 @@ const FocusSearchListCommand: ICommandAction = {
 };
 MenuRegistry.addCommand(FocusSearchListCommand);
 
-const searchInFolderCommand: ICommandHandler = (accessor, resource?: URI) => {
+const resolveResourcesForSearchIncludes = (resources: URI[], contextService: IWorkspaceContextService): string[] => {
```

Comment:
So this is code that does URI -> special search view format, and QueryBuilder does the opposite parsing, I think this code should live next to that, it could just be a function export from QueryBuilder